app/upgrades/v5/upgrades.go

```diff
@@ -99,3 +106,50 @@ func MigrateGenesis(appState genutiltypes.AppMap, clientCtx client.Context) genu
 
 	return appState
 }
+
+// ResolveAirdrop will iterate over all the available claim records and
+// attempt to swap claimed actions and unclaimed actions.
```

Comment:
describe function in the present tense

// ResolveAirdrop iterates over all the available claim records and
// attempts to swap claimed actions and unclaimed actions.
// The following priority is considered, and only one swap is performed: